lib/salesforce_logout_app.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Handles Salesforce logouts when user session is timed out.
+class SalesforceLogoutApp < Devise::FailureApp
```

Comment:
I would like to ignore the diff coverage on this unless someone knows how to effectively test this.

Reply:
I think if could be pretty easy to just test that it does what it says it does, right? Check out tests here for examples https://github.com/Exygy/sf-dahlia-web/blob/master/spec/controllers/overrides/passwords_controller_spec.rb. Maybe it's not so easy though!

What's warden?

Reply:
hmm @software-project do you have an idea on how we can test this because it's easily "callable"?

Reply:
@akegan I can't find an easy way to test for this without doing an e2e test and having them wait 15 mins for the user session to timeout. Apparently, controller tests won't really work for this -- https://github.com/plataformatec/devise/issues/3409. We can see if @software-project has something to share.